scripts/boot

```diff
@@ -141,16 +144,21 @@ if args.config:
     if VERB: print INFO, "Using config file", config
 
 hyper_name = "qemu"
-if args.solo5:
-    hyper_name = "solo5"
+if args.solo5_hvt:
+    hyper_name = "solo5-hvt"
     os.environ['PLATFORM'] = "x86_solo5"
-
-    # We expect these to be added to path now, e.g. by conan virtualenv
-    solo5_hvt  = "solo5-hvt"
-    solo5_ifup = "solo5-ifup.sh"
+    solo5_hvt = INCLUDEOS_PREFIX + "/x86_64/bin/solo5-hvt"
```

Comment:
solo5-hvt is in your binary path when its included in conan and you run activate.sh (this is needed for vmrunner and boot to work in general)